osu.Game.Rulesets.Mania/Objects/Drawables/DrawableManiaHitObject.cs

```diff
@@ -41,7 +51,7 @@ protected override float SamplePlaybackPosition
         /// Whether this <see cref="DrawableManiaHitObject"/> can be hit, given a time value.
         /// If non-null, judgements will be ignored whilst the function returns false.
         /// </summary>
-        public Func<DrawableHitObject, double, bool> CheckHittable;
+        public Func<DrawableHitObject, double, bool>? CheckHittable;
```

Comment:
and stuf like this i dunno what has happened

Reply:
Attempt to convert a single file to `#nullable true`